rollup.config.ts

```diff
@@ -24,7 +24,7 @@ export default {
     // Allow json resolution
     json(),
     // Compile TypeScript files
-    typescript({ useTsconfigDeclarationDir: true }),
```

Comment:
don't bother putting all types into the same directory, this allows typescript to resolve types properly without additional build config from user